documentation/providers/loopia.md

```diff
@@ -0,0 +1,155 @@
+Loopia is an absolutely 💩 provider of DNS API, if you want to use a decent API. 
+
+If you're SOL, and already stuck with them, then hopefully this plugin will
+save you. 
+
+What it will not save you from is the 💩iness of this provider.
+At 60 calls per minute to the API, you will have to wait to make any amount
+of changes. The http proxy does not throttle the rates. You will not receive a single
+error from there. The errors propagate from the back-end, with no headers, or
+Retry-After or anything useful. Some testing has revealed weird behaviour with NS
+records, like the control panel and back-end try to be "useful" somewhere but
+there are about eleventy million edge-cases that they haven't fixed.
+
+
+Their support is not very helpful should you encounter errors with DNS or their
+control panel that you know are errors or broken spec. 
+You may have to tell them to do their job.
+
+That said, they provide DNS. Both as a registrar, and a provider. Their prices
+are not very friendly or competitive. They provide support in English and 
+other regional variants (Norwegian, Serbian, Swedish).
+
+There are no guarantees of idempotency from their API.
+
+This plugin is based on 💩 API documents found at 
+[https://www.loopia.com/api/](https://www.loopia.com/api/)
+and by observing API responses. Hat tip to github @hazzeh whose code for the
+LEGO Loopia implementation was helpful.
+
+## Unimplemented API methods
+
+ * `removeDomain` is not implemented for safety reasons. Should you wish to remove
+a domain, do so from the Loopia control panel.
+ * `addDomain`
+ * `transferDomain` (to Loopia)
+
+This effectively means that this plugin does not access registrar functions.
+
+## Configuration
+
+To use this provider, add an entry to `creds.json` with `TYPE` set to `LOOPIA`
+along with your Loopia API login credentials.
+
+Example:
+
+```json
```

Comment:
Can you also apply the GitBook code syntax? See https://github.com/StackExchange/dnscontrol/pull/2119 and https://github.com/StackExchange/dnscontrol/pull/2137

Reply:
Think I understood - tags like these? `{% code title="dnsconfig.js" %}`?